ui/src/main/webapp/src/app/components/project-list/project-list.component.html

```diff
@@ -26,7 +26,7 @@ <h1 i18n="Project page header">Projects</h1>
         </div>
 
         <div *ngIf="projects?.length == 0" class="no-matches">
-            The current filter is hiding all projects. <a action (click)="updateSearch('')">Clear filter</a>
+            <template i18n>The current filter is hiding all projects.</template> <a action (click)="updateSearch('')"><template i18n>Remove filter</template></a>
         </div>
```

Comment:
Clear filter text was taken from OpenShift, but if you prefer remove... It doesn't really matter.